src/Parser.js

```diff
@@ -996,8 +996,11 @@ export default class Parser {
             }
             symbol = new ParseNode(symbols[this.mode][text].group,
                             text, this.mode, nucleus);
-        } else if (supportedCodepoint(text.charCodeAt(0)) &&
-            (this.mode === "text" || this.settings.unicodeTextInMathMode)) {
+        } else if ((this.settings.allowAllSymbols &&
+                    text.charCodeAt(0) >= 0x80) ||
```

Comment:
The `>= 0x80` check is to prevent things like `^` from getting interpreted as a character.  I'm open to other suggestions here.  (I thought about having a table of special symbols...)  But it seems unlikely that there are other interesting characters in the `0`-`0x7f` range that we don't already support.